### ISSUE TITLE
http/3 does not in Windows 11

### ISSUE THREAD

**Issue by m7vv** [alexandrehtrb/Pororoca#13]
I have installed version 1.6.0 and tried to send http/3 request to https://http3check.net/
 and get this text 

My system: Windows 11 (10.0.22621) 

```
System.Net.Http.HttpRequestException: Requesting HTTP version 3.0 with version policy RequestVersionExact while unable to establish 
HTTP/3 connection.
   at System.Net.Http.HttpConnectionPool.ThrowGetVersionException(HttpRequestMessage request, Int32 desiredVersion)
   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.DecompressionHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)
   at Pororoca.Infrastructure.Features.Requester.PororocaRequester.RequestAsync(IPororocaVariableResolver variableResolver, PororocaHttpRequest req, Boolean disableSslVerification, CancellationToken cancellationToken) in C:\Projetos\Pororoca\src\Pororoca.Infrastructure\Features\Requester\PororocaRequester.cs:line 37
```

**Comment by alexandrehtrb** [comment]
Hello @m7vv ,

First, thanks for reporting this issue, I had no idea this problem was happening.

There are two problems here:

1) There is a bug that HTTP/3 request are not being accomplished, I will fix it in the next release.

2) The website https://http3check.net/ does not seem to be a reliable way for checking HTTP/3 support. I use the following websites to check that:

https://cloudflare-quic.com/

https://quic.aiortc.org/

There is also a command-line tool that checks HTTP/3 availability:

https://github.com/microsoft/quicreach

**Comment by alexandrehtrb** [comment]
Solved on release 2.0.0!
